client/my-sites/plans-features-main/index.jsx

```diff
@@ -50,7 +51,14 @@ class PlansFeaturesMain extends Component {
 		);
 	}
 
-	getPlanFeatures( site, intervalType ) {
+	getPlanFeatures( params ) {
+		const {
+			site,
+			intervalType,
+			upgradePlan,
+			hideFreePlan
+		} = params;
```

Comment:
;) I suppose I should have just pulled it from props before too, or moved the function out of the class if I kept the args.